### DISCUSSION TITLE
What time in the night is the daily data ready for downloading ?

### DISCUSSION THREAD

**Discussion by ningji** [ranaroussi/yfinance#1800] [Q&A]
There was once i downloaded the daily data at 9pm, that day's volume was off a lot.
Then i tried again the next morning, it downloads the  day's data again and volume looks much better.

So to get accurate data, what time in the night should i start to download ? Thx !

**Comment by ValueRaider** [comment]
I have insight because my project `yfinance-cache` requires answer.

I find most daily volumes stop updating within 2 hours after next day market open. But not all, and in rare cases some update many days after.
